src/material/core/focus-indicators/_focus-indicators.scss

```diff
@@ -7,19 +7,43 @@
 ///   .my-app {
 ///     @include mat-strong-focus-indicators($config);
 ///   }
+
+// Mixin that handles null values and defines CSS variables accordingly
+@mixin _create-variable($name, $value) {
+  @if ($value) {
+    --#{$name}: #{$value};
+  }
+}
+
+// Mixin to dedup CSS Variables for the strong-focus-indicators mixin
+@mixin _strong-focus-indicators-helper($config) {
```

Comment:
I was imagining that we'd share this mixin too since it's only the name prefix that changes.

Reply:
Synced on this! Stored these mixins as `private-...` mixins in `src/material/core/style/_focus-indicators.scss`.